hypertools/tools/normalize.py

```diff
@@ -52,21 +54,18 @@ def normalize(x, normalize='across', internal=False, format_data=True):
         if format_data:
             x = formatter(x, ppca=True)
 
-        zscore = lambda X,y: (y - np.mean(X)) / np.std(X) if len(set(y))>1 else np.zeros(y.shape)
+        zscore = lambda X, y: (y - np.mean(X)) / np.std(X) if len(set(y)) > 1 else np.zeros(y.shape)
 
-        if normalize=='across':
+        if normalize == 'across':
             x_stacked=np.vstack(x)
             normalized_x = [np.array([zscore(x_stacked[:,j], i[:,j]) for j in range(i.shape[1])]).T for i in x]
 
-        elif normalize=='within':
+        elif normalize == 'within':
             normalized_x = [np.array([zscore(i[:,j], i[:,j]) for j in range(i.shape[1])]).T for i in x]
 
-        elif normalize=='row':
+        elif normalize == 'row':
             normalized_x = [np.array([zscore(i[j,:], i[j,:]) for j in range(i.shape[0])]) for i in x]
 
-        elif normalize==False:
```

Comment:
Nice catch! Although we want to be able to bypass normalization and return unnormalized data if `normalize=None`, so i think this bit should stay, but read:
```
if not normalize:
    normalized_x = x
```
and then the docstring should also remove mention of False